### ISSUE TITLE
CI test failure

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#1229]
Occured in #1227:

```
07:14:54 + '[' 0 -ne 0 ']'
07:14:54 + kubectl exec -t reviews-v1-2690276606-dwdjs wget -- --connect-timeout=5 --tries=1 ratings:9080
07:14:54 --2017-07-29 07:14:53--  http://ratings:9080/
07:15:15 Resolving ratings (ratings)... failed: Temporary failure in name resolution.
07:15:15 wget: unable to resolve host address 'ratings'
07:15:15 + '[' 4 -ne 0 ']'
07:15:15 + abort 'Error: could not connect from reviews-v1 to ratings:9080 service'
07:15:15 + set +x
07:15:15 ------------------------------------------------------------------------
07:15:15                           K8s Test Failed
07:15:15 Error: could not connect from reviews-v1 to ratings:9080 service
07:15:15 
07:15:15 ------------------------------------------------------------------------
07:15:15 ------------------------------------------------------------------------
07:15:15                             Cilium logs
```

**Comment by tgraf** [comment]
This is most likely a temporary failure but it would be interesting to know why it happens and whether we have additional race conditions in the tests.

**Comment by ianvernon** [comment]
Looks like this was fixed by #1400. @tgraf is there anything else you want me to do here for this ticket? I can look for race conditions in the test if you want.

**Comment by ianvernon** [comment]
I'm closing this as we have had considerable changes in the CI since this was filed, and that the CI has been stable.
